packages/r2frida/build.sh

```diff
@@ -0,0 +1,36 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/nowsecure/r2frida
+TERMUX_PKG_DESCRIPTION="Frida IO plugin for radare2"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="5.8.6"
+TERMUX_PKG_SRCURL=https://github.com/nowsecure/r2frida/archive/$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=c65979581dc514e58639b5c9492ed21780370446fad6d8cfcaaffe25e86366f1
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_DEPENDS="radare2"
+TERMUX_PKG_BUILD_DEPENDS="radare2"
```

Comment:
r2frida runs `r2` to generate an include file with the compiled form of the agent script, so it should be required

Reply:
Yeah, but PKG_BUILD_DEPENDS installs termux's r2, which is not required when cross-compiling (it will be installed in any case since radare2 is in PKG_DEPENDS also)